deploy/cw-bundle.yaml

```diff
@@ -12138,7 +12161,7 @@ spec:
           value: percona-xtradb-cluster-operator
         - name: DISABLE_TELEMETRY
           value: "false"
-        image: perconalab/percona-xtradb-cluster-operator:main
+        image: perconalab/percona-xtradb-cluster-operator:feat-add-generated-secrets-options
```

Comment:
The changes to the image should be reverted to `main` across all the similar occurrences.